backend/src/main/java/de/amos/apachepulsarui/controller/TopicController.java

```diff
@@ -52,6 +51,17 @@ public ResponseEntity<TopicDto> getTopicWithMessagesByName(@RequestParam String
        return new ResponseEntity<>(topicService.getTopicWithMessagesByName(name), HttpStatus.OK);
     }
 
+    @GetMapping("/{topic}/subscription/{subscription}")
+    public ResponseEntity<SubscriptionDto> getSubyscriptionByNameAndTopic(@PathVariable String topic, @PathVariable String subscription) {
+        return new ResponseEntity<>(topicService.getSubscriptionByTopic(topic, subscription), HttpStatus.OK);
+    }
+
+    @GetMapping("{topic}/producer/{producer}")
+    public ResponseEntity<ProducerDto> getProducerByNameAndTopic(@PathVariable String topic, @PathVariable String producer) {
+        return new ResponseEntity<>(topicService.getProducerByTopic(topic, producer), HttpStatus.OK);
+    }
```

Comment:
We need to use @RequestParam here, because the topicnae won't work as a pathparameter